tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/TagMap.java

```diff
@@ -114,15 +113,20 @@ private static String[] toArray(Map<String, String> data) {
         return values;
     }
 
-    /** Returns a new {@link TagMap} with on additional entry. */
+    /** Returns a new {@link TagMap} with on additional or updated entry. */
```

Comment:
nit:
```suggestion
    /** Returns a new {@link TagMap} with one additional or updated entry. */
```

Reply:
Thanks! I think I meant "an" here
```suggestion
    /** Returns a new {@link TagMap} with an additional or updated entry. */
```